storage/storage_local.go

```diff
@@ -31,26 +31,70 @@ func NewStorageLocal(path string) Storage {
 }
 
 // DownloadArtifactToStorage : Download the artifact from the URI to Storage
-func (s storageLocal) DownloadArtifactToStorage(uri string, id Identifier) {
+func (s storageLocal) DownloadArtifactToStorage(uri string, id Identifier, codes ...int) (int, error) {
```

Comment:
Seems worth calling out exactly what codes corresponds to here as it seems a bit non-obvious to me.

Reply:
My intent with this is to allow acceptance of more than just a 200 from a proxy server depending on the use case. I thought I had one for a while (thus putting in the functionality) but it turned out it was sending an error in the payload instead of the artefact. Instead of removing the functionality (which I could totally do) I thought I'd leave it in.

Reply:
Sorry, should have been clearer, makes sense to keep the functionality in, meant that I feel it's worth clarifying the variable name (`acceptableResponseCodes` maybe?) or adding a doc string to make it a bit clearer.